google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -310,7 +310,7 @@ public void addSupport(@NotNull Module module,
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
 
-      // Called on New Project or New Module dialogs.
+      // Called on New Project, New Module or Add Framework Support dialogs.
```

Comment:
how about: 'Called when creating a new App Engine module from the 'new project' or 'new module' wizards or upon adding App Engine 'Framework Support' to an existing module'